frontend/src/views/OtherUses/_schema.jsx

```diff
@@ -139,6 +240,41 @@ export const otherUsesColDefs = (optionsData, errors) => [
       params.value || <Typography variant="body4">Select</Typography>,
     cellStyle: (params) => StandardCellErrors(params, errors)
   },
+  {
+    field: 'ciOfFuel',
+    headerName: i18n.t('otherUses:otherUsesColLabels.ciOfFuel'),
+    valueFormatter: (params) => parseFloat(params.value).toFixed(2),
+    cellStyle: (params) => StandardCellErrors(params, errors),
+    editable: false,
+    valueGetter: (params) => {
```

Comment:
because there are dependencies based on the selection of the user. This is related to the comment above:
Considering the following object:
```
{
    "fuelTypeId": 1,
    "fuelType": "Biodiesel",
    "fossilDerived": null,
    "provision1Id": null,
    "provision2Id": null,
    "defaultCarbonIntensity": 100.21,
    "fuelCodes": [
        {
            "fuelCodeId": 2,
            "fuelCode": "BCLCF124.4",
            "carbonIntensity": 3.62
        },
        {
            "fuelCodeId": 3,
            "fuelCode": "BCLCF125.4",
            "carbonIntensity": -2.14
        },
        {
            "fuelCodeId": 4,
            "fuelCode": "BCLCF138.5",
            "carbonIntensity": 4.26
        },
        {
            "fuelCodeId": 6,
            "fuelCode": "BCLCF251.2",
            "carbonIntensity": 0.35
        },
        {
            "fuelCodeId": 8,
            "fuelCode": "BCLCF274.2",
            "carbonIntensity": -15.03
        },
        {
            "fuelCodeId": 15,
            "fuelCode": "BCLCF362.1",
            "carbonIntensity": -1
        }
    ],
    "provisionOfTheAct": [
        {
            "provisionOfTheActId": 2,
            "name": "Fuel code - section 19 (b) (i)"
        },
        {
            "provisionOfTheActId": 3,
            "name": "Default carbon intensity - section 19 (b) (ii)"
        }
    ],
    "units": "L"
}
```
We are just checking if provision_of_the_act is 'Fuel code - section 19 (b) (i)', If it is the case, then we find the fuel_code associated to the fuel_type selected and provide the value to ciOfFuel. Otherwise, just take the value from defaultCarbonIntensity in fuel_type. which in this case is  100.21.

Reply:
I need to see how the ciOfFuel changes when the type of fuel and fuel code change, I cannot wait to save and retrieve the information from the response of the creation.

Reply:
We talked about yesterday and how it should be moved to the backend.

Reply:
Yes, the example you showed related to Fuell Supply was one liner because it is assigning default ciOfFuel. In this case, I need to check how the ciOfFuel changes based on how the user selects the fuel type and fuel code.